ansible/roles/operator-pipeline/templates/openshift/openshift-pipeline-sa-scc-role-bindings.yml

```
@@ -2,7 +2,7 @@
 kind: ClusterRoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
-  name: "system:openshift:scc:pipelines-custom-scc"
+  name: "pipeline:pipelines-custom-scc:{{ oc_namespace }}"
```

Can we use a more consistent naming scheme? If we want to match the role binding for the admin service account, this would become something like:
```suggestion
  name: "pipeline-{{ oc_namespace }}-pipelines-custom-scc"
```

---

To add to that, IIRC the oc namespaces are formatted "operator-pipeline-<env>", so there'll be 3 different mentions of"pipeline" all close together. Is that a bit much? Maybe just something like "{{ oc_namespace }}-custom-scc"?

---

yep, changed